pkg/api/defaults-kubelet.go

```diff
@@ -58,6 +58,7 @@ func (cs *ContainerService) setKubeletConfig() {
 	staticWindowsKubeletConfig["--image-pull-progress-deadline"] = "20m"
 	staticWindowsKubeletConfig["--resolv-conf"] = "\"\"\"\""
 	staticWindowsKubeletConfig["--eviction-hard"] = "\"\"\"\""
+	staticWindowsKubeletConfig["--protect-kernel-defaults"] = ""
```

Comment:
@andyzhangx FYI the above pattern is the current way to explicitly remove kubelet flags for Windows nodes. See the `removeKubeletFlags` func below, which will eventually process `""` vals by removing the keys entirely.